src/Components/Checkout.jsx

```diff
@@ -0,0 +1,35 @@
+import { useState } from "react";
+import Swal from 'sweetalert2'
+import { cartContext } from "../Context/CartProvide"; 
+import { useContext} from "react";
+function Checkout(){
+    let [isCheckout, setIsCheckout] = useState(false);
+    let {setCart} = useContext(cartContext);
+    function handleSubmit(e){
+        e.preventDefault();
+        setIsCheckout(false);
+        setCart([])
+        Swal.fire({
+            title: 'Sent',
+            text: 'Your order was sent successfully',
+            icon: 'success',
+            confirmButtonText: 'OK',
+            confirmButtonColor:"#99dca8"
+        })
+    }
+    return(
+       <div className="checkout">
+           {  
+           isCheckout?
+            <form onSubmit={(e)=>{handleSubmit(e)}}>
```

Comment:
```suggestion
            <form onSubmit={handleSubmit}> 
```
use point free style in such cases